di-ipv-queue-stub/enqueue-event/src/services/queueService.ts

```diff
@@ -108,3 +110,17 @@ export const dequeueEvent = async (
     return null;
   }
 };
+
+export const findBranchQueues = async (): Promise<string[]> => {
+  const result = await sqsClient.send(new ListQueuesCommand({
+    QueueNamePrefix: "stubQueue_branch_",
```

Comment:
This is limited for now to avoid deleting queues that people might need... In future we might want to expand this.

In particular, we don't want to delete queues linked to real environments (i.e. build or individual dev environments)